docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -184,40 +184,40 @@ function setData(
 ) public
 ```
 
-Sets array of data at multiple keys in the account storage.
+Sets an array of data at multiple keys in the account storage.
 
-_Triggers the **[DataChanged](#datachanged)** event when setting data successfully._
+_Triggers the **[DataChanged](#datachanged)** event when successfully setting the data._
 
 :::note
-**The `setData(...)` function can only be called by the current owner of the contract.**
+The `setData(...)` function can only be called by the current owner of the contract.
 :::
 
 #### Parameters:
 
-| Name     | Type       | Description                     |
-| :------- | :--------- | :------------------------------ |
-| `keys`   | bytes32[ ] | The keys for which to set data. |
-| `values` | bytes[ ]   | The array of data to set.       |
+| Name     | Type            | Description                     |
+| :------- | :-------------- | :------------------------------ |
+| `keys`   | bytes32[&nbsp;] | The keys for which to set data. |
+| `values` | bytes[&nbsp;]   | The array of data to set.       |
 
 ### getData (Array)
 
 ```solidity
 function getData(bytes32[] memory keys) public view returns (bytes[] memory values)
 ```
 
-Retrieve an array of data for multiple given keys.
+Retrieves an array of data for multiple given keys.
```

Comment:
I find this style and sentence perfect 👍 
This should be used as a documented example for the conventions in the docs!

Reply:
thank you!